common/tests/util.py

```diff
@@ -118,6 +120,50 @@ def make_non_duplicate_record(factory, identifying_fields=None):
     return not_duplicate
 
 
+def assert_records_match(expected: TrackedModel, imported: TrackedModel, ignore=set()):
+    """
+    Asserts that every value for every field in the imported model is the same
+    as the data in the expected model.
+
+    System fields that will change from model to model are not checked. Any
+    field names given to `ignore` will also not be checked.
+    """
+    checked_fields = (
+        set(field.name for field in imported._meta.fields)
+        - set(field.name for field in TrackedModel._meta.fields)
+        - {"trackedmodel_ptr"}
+        - ignore
+    )
+
+    for field in checked_fields:
+        imported_value = getattr(imported, field)
+        source_value = getattr(expected, field)
+        assert (
+            imported_value == source_value
+        ), f"imported '{field}' ({imported_value} - {type(imported_value)}) does not match source '{field}' ({source_value} - {type(source_value)})"
+
+
+def assert_many_records_match(
+    expected: Sequence[TrackedModel],
+    imported: Sequence[TrackedModel],
+    ignore=set(),
```

Comment:
Annoyingly, python will persist classes (like set() here) called in the constructor, which isn't usually what you want.

Reply:
I thought sets were immutable, but they aren't – I should be using `frozenset`.
```suggestion
    ignore=frozenset(),
```

Reply:
Nice... they should have frozen versions of everything... since python is gradually turning into Perl5, maybe they will come with some weird new syntax for it...  `ignore=⛄set()`

Reply:
🤢